src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/RedNodes/SyntaxFactsWriter.vb

```diff
@@ -86,6 +86,10 @@ Public Class SyntaxFactsWriter
                     tokenText = tokenText.Replace("""", """""")
                 End If
 
+                If tokenText IsNot Nothing AndAlso tokenText.Contains(vbCrLf) Then
```

Comment:
`LineContinuationTrivia` encodes a CRLF into it's `TokenText`. We replace this with whatever the environment newline is.